path_oidc.go

```diff
@@ -44,6 +58,107 @@ func pathOIDC(b *jwtAuthBackend) []*framework.Path {
 	}
 }
 
+func (b *jwtAuthBackend) pathCallback(ctx context.Context, req *logical.Request, d *framework.FieldData) (*logical.Response, error) {
+	state := b.verifyState(d.Raw["state"].(string))
```

Comment:
What would happen here if the state weren't a string or weren't provided?

Reply:
Bad things 😛. Will fix...

Reply:
Refactored to use the Get() method which will ensure we'll at least always have a string.